pkg/statsd/aggregator.go

```diff
@@ -76,99 +78,102 @@ func (a *MetricAggregator) Flush(flushInterval time.Duration) {
 	})
 
 	a.metricMap.Timers.Each(func(key, tagsKey string, timer gostatsd.Timer) {
-		if count := len(timer.Values); count > 0 {
-			sort.Float64s(timer.Values)
-			timer.Min = timer.Values[0]
-			timer.Max = timer.Values[count-1]
-			n := len(timer.Values)
-			count := float64(n)
+		if hasHistogramTag(timer) {
```

Comment:
As pointed out in another comment, this can be restructured a bit to make the diff a lot cleaner (I think the suggested diff might be broken, sorry if it's not clear):
```suggestion
		if hasHistogramTag(timer) {
			timer.Histogram = latencyHistogram(timer, a.histogramLimit)
			a.metricMap.Timers[key][tagsKey] = timer // line needs to be done on both sides of the condition, because it's not a pointer
			return
		}
		... original code with original indenting ...
		a.metricMap.Timers[key][tagsKey] = timer // line needs to be done on both sides of the condition, because it's not a pointer
```

There's a few other places where similar could be done, but those diffs are fairly clean compared to this one (I think the line breaks confuse githubs diff algo), so only this one really needs it.